contracts/base/LoansBase.sol

```diff
@@ -227,12 +227,17 @@ contract LoansBase is LoansInterface, Base, SettingsConsts {
 
         loans[loanID].status = ZeroCollateralCommon.LoanStatus.Active;
 
-        // give the recipient their requested amount of tokens
-        if (loans[loanID].loanTerms.recipient != address(0)) {
-            lendingPool.createLoan(amountBorrow, loans[loanID].loanTerms.recipient);
-        } else {
-            lendingPool.createLoan(amountBorrow, loans[loanID].loanTerms.borrower);
-        }
+        address escrow = settings.getEscrowFactory().createEscrow(loanID);
+
+//        // give the recipient their requested amount of tokens
+//        if (loans[loanID].loanTerms.recipient != address(0)) {
+//            lendingPool.createLoan(amountBorrow, loans[loanID].loanTerms.recipient);
+//        } else {
+//            lendingPool.createLoan(amountBorrow, loans[loanID].loanTerms.borrower);
+//        }
+
+        // NOTE: Only send loan to escrow contract for now
+        lendingPool.createLoan(amountBorrow, escrow);
```

Comment:
We are sending the funds to the escrow contract. So at this point, what can the borrower do with the funds?

Please, remove the commented lines.

Reply:
Commented lines removed